data/channels/badmouseproductions.yml

```diff
@@ -6,8 +6,10 @@ providers:
     name: BadMouseProductions
     slug: xaxie1
     url: https://www.youtube.com/xaxie1
-    description: ""
-    subscribers: 64969
+    description: |-
```

Comment:
Scuffed as symbols
also
For consistency should this content be surrounded by quotation marks?